util/collate/pinyin_tidb_as_cs.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collate
+
+type zhPinyinTiDBASCS struct {
```

Comment:
What does 'ASCS' mean exactly?

Reply:
I rename it to `zhPinyinTiDBASCSCollator` and add comments. 'ASCS' for accent-sensitive and case-sensitive